src/Symfony/Component/Messenger/Bridge/Redis/Tests/Transport/RedisExtIntegrationTest.php

```diff
@@ -450,7 +450,7 @@ private function getConnectionStream(Connection $connection): string
     private function skipIfRedisClusterUnavailable()
     {
         try {
-            new \RedisCluster(null, explode(' ', getenv('REDIS_CLUSTER_HOSTS')));
+            new \RedisCluster(null, getenv('REDIS_CLUSTER_HOST') ? explode(' ', getenv('REDIS_CLUSTER_HOST')) : []);
```

Comment:
we would pass `['']` otherwise which let's the constructor of the `RedisCluster` class emit a warning (see https://github.com/symfony/symfony/actions/runs/14302806931/job/40080130992#step:9:676):

> `RedisCluster::__construct(): Seed '' not in host:port format, ignoring`